core-initrd/latest/factory/usr/lib/systemd/system/plymouth-start.service.d/core-override.conf

```diff
@@ -1,6 +1,4 @@
-[Unit]
-After=snap-initramfs-mounts.service
-
 [Service]
 ExecStart=
 ExecStart=/usr/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session --ignore-serial-consoles
+ExecReload=-/usr/bin/plymouth reload
```

Comment:
Do we need this line? (we are not calling `systemctl reload` anymore, although maybe it is fine to leave around)

Reply:
This could have stayed. But yes, it is useless. I will remove it.